src/rules/selector-no-empty/__tests__/index.js

```diff
@@ -0,0 +1,46 @@
+import {
+  messages,
+  ruleName,
+} from ".."
+import rules from "../../../rules"
+import { testRule } from "../../../testUtils"
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+
+  accept: [ {
+    code: ".a, .b {color: red}",
+  }, {
+    code: " a {color: red;}",
+  } ],
+
+  reject: [ {
+    code: "   {color: red;}",
+    message: messages.rejected,
+    line: 1,
+    column: 4,
+  }, {
+    code: "   ,  .a   , .b {}",
+    message: messages.rejected,
+    line: 1,
+    column: 4,
+  }, {
+    code: ".a,  ,.b {}",
+    message: messages.rejected,
+    line: 1,
+    column: 6,
+  }, {
+    code: ".a,    .b, {}",
+    message: messages.rejected,
+    line: 1,
+    column: 11,
+  }, {
+    code: "   .a,    .b, {}",
+    message: messages.rejected,
+    line: 1,
+    column: 14,
+  } ],
+})
```

Comment:
Can we get one or two tests for the other parsers please e.g. `sugarss`, `less` and `scss`? I'm curious to seem, especially `sugarss` parse empty selectors in the same way.